### PR TITLE
Positioning and floating project completed

### PR DESCRIPTION
**Pull request by voscarmv**
We completed the third HTML/CSS project. Waiting to hear from the TSEs about our review.

### CODE REVIEW

**Comment by addod19** [comment]
## Pull Request under **review** by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,453 @@
...
+            <button class="button-noborder"><span><img class="icons-buttons" src="media/icons8-google-blog-search-50.png" alt="share-icon"></span></button>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main>
+      <div>
+        <section>
+          <h2 class="void-heading">.</h2>
+          <div class="article-text-section">
```
Comment (addod19) [inline index.html:48]: Let us improve semantics by using ```<article>``` instead of ```<div class="article-text-section">```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi Team, good job on the project so far 👍 . There are a few changes to get this project approved. Find comments below: 😄 

- Kindly add a link to NY times logo.
ny
- Kindly center the text as described below:
center
main

- In your next project, remember to add a title and a summary in your pull request.
pr

Kindly make these changes and submit code review.
Good luck 😄 
Happy coding 💻

**Comment by Salvador-ON** [comment]
we added link to logo

we centered the title

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved 🟢

Hi Team, **congratulations** implementing all the requested changes. 💯 
You can now merge to master and submit project completion form. 🍾 

Remember to add a title and short summary for your pull request in your next project.

Good luck in your next project.
Happy coding 💻

**Pull request merged**